lang/stack/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github                      1.0
+
+github.setup        commercialhaskell stack 2.1.1
+
+name                stack
+categories          lang haskell
+maintainers         nomaintainer
```

Comment:
You don't want to be the maintainer?

Reply:
I’ll be the maintainer.

Reply:
the maintainer line should follow: ```{example.org:julesverne @github-handle} openmaintainer``` and unless there is a good reason not to, adding "openmaintainer" to allow others to make "minor changes" without your approval would be preferred

Reply:
Done. Thanks.